requirements.txt

```diff
@@ -1,4 +1,4 @@
-pyyaml~=6.0
+pyyaml>=5.0
```

Comment:
This dependency clashes with docker-compose's, which asks for version 5. It does not make sense that testing dependencies define what the application can use, but I have not been able to solve this other than to lower the library requirement. If anyone knows of another solution, please tell.